src/main/java/io/codelex/enums/practice/Exercise2/Game.java

```diff
@@ -0,0 +1,79 @@
+package io.codelex.enums.practice.Exercise2;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Game {
+    private static int playerWonCounter = 0;
+    private static int computerWonCounter = 0;
```

Comment:
class scope primites ir default value

Reply:
https://www.tutorialspoint.com/default-value-of-primitive-data-types-in-java

bet tas ir tikai class scope!! iekš metodēm jāpiešķir 0, pats tieši nesen to arī iemācījos 😄